endpointmanager/pkg/fhirendpointquerier/populatefhirendpoints.go

```diff
@@ -75,6 +76,9 @@ func formatToFHIREndpt(endpoint *fetcher.EndpointEntry) (*endpointmanager.FHIREn
 		uri = uri + "/"
 	}
 
+	splitEndpoint := strings.Split(uri, "://")
+	uri = "https://" + splitEndpoint[len(splitEndpoint)-1]
```

Comment:
Based on the conversation with Jeff in Slack, I think what we should do here is instead of always tacking on `https://`, to instead use toLower() if the http[s] exists, to at least catch the HTTPS vs https issue, and tack it on if it doesn't exist. This won't catch the issue where you have the same URL where the only difference is http/https, but I think it's probably better than messing with an endpoint where it only works for http and we accidentally break it.

Reply:
I agree, I think this makes more sense!